README.md

```diff
@@ -72,8 +75,9 @@ will have to:
 
 1. Create a parameter table
 
-If you are using Python, some handy functions of the PEtab library can help 
-you with that. This include also a PEtab validator called `petablint.py` which
+If you are using Python, some handy functions of the
+[PEtab library](https://petab.readthedocs.io/en/latest/modules.html) can help
+you with that. This include also a PEtab validator called `petablint` which
 you can use to check if your files adhere to the PEtab standard. If you have 
```

Comment:
Not sure what you mean. petablint works well with yaml files. just that not everything is checked in full detail, and for rather complex cases, it will tell you that it can't handle that. I'd consider that too much information for here. the explanation what it cannot do, is probably way too confusing for a person just stumbling over this readme file.

Reply:
you might be right about it being more confusing at this point